### PR TITLE
Fix input association in alive translation

### PR DESCRIPTION
**Pull request by manasij7479**

### CODE REVIEW

File: include/souper/Infer/AliveDriver.h [hunk 1]
```diff
@@ -44,6 +44,8 @@ class AliveDriver {
   Inst *LHS, *PreCondition;
 
   Cache LExprCache, RExprCache;
+  std::map<const Inst *, std::string> NameMap;
+  void copyInputs(Cache& From, Cache &To, IR::Function &RHS);
```
Comment (regehr) [inline include/souper/Infer/AliveDriver.h]: not that big of a deal, but you need to consistently put the & (or *) next to the name, not next to the type

File: lib/Infer/AliveDriver.cpp [hunk 2]
```diff
@@ -433,10 +433,23 @@ souper::AliveDriver::synthesizeConstantsWithCegis(souper::Inst *RHS, InstContext
   return ConstMap;
 }
 
+void souper::AliveDriver::copyInputs(souper::AliveDriver::Cache& From,
+                                         souper::AliveDriver::Cache& To,
```
Comment (regehr) [inline lib/Infer/AliveDriver.cpp]: actually just fix all these quickly, please

File: include/souper/Infer/AliveDriver.h (hunk 1 above)
Comment (manasij7479) [reply include/souper/Infer/AliveDriver.h]: fixed

File: lib/Infer/AliveDriver.cpp (hunk 2 above)
Comment (manasij7479) [reply lib/Infer/AliveDriver.cpp]: fixed

**Comment by regehr** [comment]
LGTM thanks!

**Pull request merged**